README.md

```diff
@@ -20,7 +20,31 @@ Bow is split in multiple modules that can be consumed independently. These modul
 - `BowBrightFutures`: module to provide an integration with BrightFutures.
 - `BowRx`: module to provide an integration with RxSwift.
 
-Bow is available using Cocoapods, Carthage and Swift Package Manager.
+Bow is available using CocoaPods and Carthage.
+
+### CocoaPods
+
+You can consume each Bow module as a separate pod. You can add these lines to your Podfile at your convenience:
+
+```ruby
+pod "Bow", "~> 0.2.0"
+pod "BowOptics", "~> 0.2.0"
+pod "BowRecursionSchemes", "~> 0.2.0"
+pod "BowFree", "~> 0.2.0"
+pod "BowGeneric", "~> 0.2.0"
+pod "BowResult", "~> 0.2.0"
+pod "BowEffects", "~> 0.2.0"
+pod "BowRx", "~> 0.2.0"
+pod "BowBrightFutures", "~> 0.2.0"
+```
+
+### Carthage
+
+Carthage with download the whole Bow project, but it will compile individual frameworks for each module that you can use separately. Add this line to your Cartfile:
```

Comment:
Typo, `Carthage **will** download`

Reply:
```suggestion
Carthage will download the whole Bow project, but it will compile individual frameworks for each module that you can use separately. Add this line to your Cartfile:
```